### DISCUSSION TITLE
Tap to augment in WebXR

### DISCUSSION THREAD

**Discussion by HimanshuBarak** [google/model-viewer#3106] [Q&A]
Hey 

By default, the model automatically augments as soon as the camera points to a surface.

 I want the user to move their camera to the desired location first and then tap on the location to augment the model in that specific location/surface. Is there any way to achieve this ? 

Thanks for any guidance.

**Comment by elalish** [comment]
No, but the user can drag the object to where they want it. We try to  keep this  as simple as possible, since just  getting ARCore to find anywhere to place the object is not easy and when it's unreliable the  users tend to get irritated. As soon as they move their phone enough to find the floor, the object will move  to the floor. You can use child HTML elements to give them instructions  and base your CSS on  our `ar-status` attribute.

**Reply by HimanshuBarak** [reply]
Yes the instructions based on ar-status were enough for my use case. 

And everything was even working fine when I tested it yesterday but now  I keep getting this error
![image](https://user-images.githubusercontent.com/66054954/149627291-e1af19b1-308f-459c-ad6e-c057137e15a6.png)

I can't figure out what is wrong.
Any help will be appreciated.

**Reply by elalish** [reply]
For security, Chrome will not activate XR unless the user clicks something to activate it. If you're getting this it  generally means you're trying to circumvent our Enter  AR button (enter automatically), but it's there for a  reason.
